src/PackagePrivate/InMemoryTermIdsAcquirer.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Wikibase\TermStore\MediaWiki\PackagePrivate;
+
+class InMemoryTermIdsAcquirer implements TermIdsAcquirer {
+
+	private $terms;
+	private $lastId = 0;
+
+	/**
+	 * @inheritDoc
+	 */
+	public function acquireTermIds( array $termsArray ): array {
+		$ids = [];
+
+		foreach ( $termsArray as $type => $termsOfType ) {
+			foreach ( $termsOfType as $lang => $terms ) {
+				$terms = (array)$terms;
+
+				foreach ( $terms as $term ) {
+					if ( isset( $this->terms[$type][$lang][$term] ) ) {
+						$ids[] = $this->terms[$type][$lang][$term];
+					} else {
+						$this->terms[$type][$lang][$term] = ++$this->lastId;
```

Comment:
This uses `$this->terms` as if it is an array, but it is never initialized. Can you please add `= []` to the property declaration?

Reply:
yes sure